src/main/java/app/controllers/OrderController.java

```diff
@@ -34,22 +44,47 @@ private static void continueRequest(Context ctx, ConnectionPool connectionPool)
         }
     }
 
-    public void sendOrderRequest(Context ctx, ConnectionPool connectionPool) {
+    public static void sendOrderRequest(Context ctx, ConnectionPool connectionPool) {
+
+        try {
+            int width = Integer.parseInt(ctx.formParam("width"));
+            int length = Integer.parseInt(ctx.formParam("length"));
+
+            String title = "Carport bredde: " + width + " cm & Carport længde: " + length + "cm";
+
+            List<OrderBillItem> orderBillItemList = OrderBillGenerator.generateOrderBill(width, length, connectionPool);
+
+            Order newOrder = new Order(-1, title, Order.OrderStatus.WAITING_FOR_REVIEW, null, orderBillItemList, LocalDateTime.now());
+
+            Customer customer = ctx.sessionAttribute("currentUser");
+            OrderMapper.createOrder(newOrder, customer.getId(), connectionPool);
+
+            ctx.redirect("/request-confirmation");
+
+        } catch (DatabaseException e) {
+            ctx.result("Error sending request." + e.getMessage());
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Remove the catching of SQL exception. Only database exceptions should be thrown from the mapper